components/Header/SecondaryNav/index.tsx

```diff
@@ -0,0 +1,28 @@
+export default function SecondaryNav(): JSX.Element {
+  return (
+    <nav aria-label="Secondary">
+      <ul className="capitalize">
+        <li>
+          <a href="#section-one" target="_self">
```

Comment:
Do we need `target` if it's set to `_self`?

Reply:
The reason is Storybook override target value in head:
```html
<head>
  <base target="_parent" />
</head>
```
Every time user clicks on a link, a new tab would pop up. I have tried [add a new base element](https://storybook.js.org/docs/react/configure/story-rendering#adding-to-head) to Storybook's head and got this:
```html
<head>
  <base target="_parent" />
  <base target="_self" />
</head>
```
But no luck, still pup up a new tab. However, add `target='_self'` is also annoying. What do you think?

Reply:
I was just wondering. But good point -- this is only for Storybook. So it's fine.

When using `target` with anything other than self, it is best for security reasons to include `rel="noopener noreferrer"`:

https://web.dev/external-anchors-use-rel-noopener/

Reply:
Worth looking at: https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/rel